browser/about_flags.cc

```diff
@@ -224,6 +225,9 @@ constexpr char kNativeBraveWalletName[] = "Enable Brave Wallet";
 constexpr char kNativeBraveWalletDescription[] =
     "Native cryptocurrency wallet support without the use of extensions";
 
+constexpr char kBraveNewsName[] = "Enable Brave News";
+constexpr char kBraveNewsDescription[] = "Enable Brave News";
```

Comment:
Should description be the same as the name?

Reply:
probably it's best if not. gave it a different description